src/xgov-dapp/src/main.tsx

```diff
@@ -58,15 +59,26 @@ const router = createBrowserRouter([
 
 const rootElement = document.getElementById('root') as HTMLElement
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 1000 * 60,
```

Comment:
same as above

Reply:
These are normally overridden by `useQuery` hooks and are the "sane defaults" which shouldn't really change.

Here is an example of when the QueryOptions are being set: 

https://github.com/algorandfoundation/nft_voting_tool/blob/feat/reusable-hooks/src/xgov-dapp/src/hooks/use-snapshot-query.ts#L21

We could add it to the ENV for the defaults but I'm not sure if there is a benefit that I can think of. Any REST request that should be different will be overridden in the implementation